src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        setColor(color);
+        obtainTheArea();
```

Comment:
It's not necessary to call 'obtainTheArea()' in the constructor. The area can be calculated when it's needed, for example in the 'getArea()' method or before calling 'draw()' if the area is not expected to change.